src/serac/physics/utilities/finite_element_state.cpp

```diff
@@ -33,6 +33,18 @@ FiniteElementState::FiniteElementState(mfem::ParMesh& mesh, mfem::ParGridFunctio
   true_vec_ = 0.0;
 }
 
+FiniteElementState::FiniteElementState(mfem::ParMesh& mesh, FiniteElementState& fe_state, const std::string& name)
+    : FiniteElementState(mesh, fe_state.gridFunc(), name)
+{
+}
+
+FiniteElementState& FiniteElementState::operator=(const double value)
+{
+  true_vec_ = value;
+  distributeSharedDofs();
```

Comment:
Does the prolongation operator here zero out the non-truedofs() or does it leave them as is? If `value` is not the same on all ranks this operation might be a little tricky to interpret?

Reply:
The prolongation operator picks up the correct true value from the owning process (it is MPI-aware).

Reply:
I was concerned that it might be confusing for a user if they set the `FiniteElementState` to a particular value (that is not the same on all ranks), and the owning process sets it to a different value. Maybe it might be worth caveating that `operator=` mainly applies to the true d.o.fs.

Reply:
I think semantically this is the right thing to do, as the "owning" process should set the DOF on non-owning processes. I'll add a not in the doxygen as you suggested.